schema-next.yaml

```diff
@@ -4,6 +4,14 @@ versions:
   next:
     metrics:
       changes:
+        # https://github.com/open-telemetry/semantic-conventions/pull/966
+        - rename_metrics:
+            db.client.connections.usage: db.client.connection.count
+            db.client.connections.idle.max: db.client.connection.idle.max
+            db.client.connections.idle.min: db.client.connection.idle.min
+            db.client.connections.max: db.client.connection.max
+            db.client.connections.pending_requests: db.client.connection.pending_requests
+            db.client.connections.timeouts: db.client.connection.timeouts
```

Comment:
these are not all deprecated metrics, it should be 9 entries according to the file above

Reply:
3 of them are not renames since the unit was changed from `ms` to `s`

Reply:
So the tooling forces me to add all previous metric names in one of the yaml files - that's why we have all 9 in [deprecated/database.yaml](https://github.com/open-telemetry/semantic-conventions/pull/966/files#diff-1d811a06d82d206d944c19019884c092fcf92747f1cf6f014f3cdcd5e1f6c1aa). As I learned in a thread above, this is a recent change, previously this wasn't enforced.

And as @trask says, 6 are just renames, the other 3 change both the unit and the metric name. I wonder if this difference should be visible in [deprecated/database.yaml](https://github.com/open-telemetry/semantic-conventions/pull/966/files#diff-1d811a06d82d206d944c19019884c092fcf92747f1cf6f014f3cdcd5e1f6c1aa). From what I know there is no syntax for just renaming there, right? Only think we can do there is to mark it with `deprecated:`.

Reply:
But they are renames as well? we do change from `db.client.connections.use_time` to `db.client.connection.use_time` and In addition to this we also do change the unit. 

In this case we shouldn't add it to the `rename_metrics` in schema?

Reply:
I looked at some of the PRs (not all of them) that added `rename_metrics` - those seem to be purely renames. If the unit changes, I'd interpret it more than a rename.

I don't have full context on how schema translation works, but my guess is that everything that is under `rename_metrics`, could be just renamed and be 100% compatible with the new schema. If so, then the ones where we change the unit should not be under `rename_metrics` - they should be kept as is and then be marked as deprecated. In theory there could be a correct translation for rename+unit change, but from what I know, there is no syntax for that and that would still be different from just renaming.

> In this case we shouldn't add it to the rename_metrics in schema?

So, unfortunately I'm not sure, so I'd defer to @trask and @lmolkova.

Reply:
We will need to  add a new "change units" schema transformation to schema files for a unit change.

cc @lmolkova @trask

Reply:
Created https://github.com/open-telemetry/opentelemetry-specification/issues/4028, I don't think we need to block this PR on this since it's an experimental metric and we don't have schema transformation in practice.

Reply:
There already existed a similar issue: https://github.com/open-telemetry/opentelemetry-specification/issues/3729

Also, we did unit changes for experimental metrics in the past (for HTTP, and for [FaaS](https://github.com/open-telemetry/semantic-conventions/pull/384)), I think it's fine as long as it's part of stabilization effort.

However, I wonder whether it should be covered by some backward compatibility mechanisms like `OTEL_SEMCONV_STABILITY_OPT_IN`, or whether implementations switching units are supposed to do it via "hard" breaking changes?